UserShiftsApiService/UserShiftsApiService/Migrations/20250128131959_AddUserDateRangeScheduleRequestEntity.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace UserShiftsApiService.Migrations
+{
+    /// <inheritdoc />
+    public partial class AddUserDateRangeScheduleRequestEntity : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "Users",
```

Comment:
nono. you cant create the table again. the migration now should change the existing table